<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/新前缀/images/6.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-radius:4px;
background-image: url(:/新前缀/images/login_bgx.gif);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请您输入用户名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-image: url(:/新前缀/images/login_bgx.gif);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-image: url(:/新前缀/images/login_bgx.gif);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>请您输入密码</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-image: url(:/新前缀/images/login_bgx.gif);</string>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images1.qrc"/>
 </resources>
 <connections/>
</ui>
